enterprise/models.py

```diff
@@ -905,6 +907,7 @@ def create_order_for_enrollment(self, course_run_id, discount_percentage):
                     "enterprise_customer_name": self.enterprise_customer.name,
                     "enterprise_customer_uuid": str(self.enterprise_customer.uuid),
                     "discount_percentage": float(discount_percentage),
+                    "sales_force_id": sales_force_id,
```

Comment:
We're adding sales_force_id to the enrollment data, but will that field be persisted? I see that we have a discussion about this in the ticket. Do we need to have a separate ticket to update ecommerce to persist the sales_force_id when creating the manual order?